libs/stokesian_dynamics/CMakeLists.txt

```diff
@@ -1,9 +1,10 @@
-if(STOKESIAN_DYNAMICS_GPU)
-  cuda_add_library(sd_gpu SHARED sd_gpu.cu)
-  target_link_libraries(sd_gpu PRIVATE
-    ${CUDA_CUBLAS_LIBRARIES}
-    ${CUDA_cusolver_LIBRARY})
-endif()
+# temporarily disabled to at least be able to test the CPU version
+# if(STOKESIAN_DYNAMICS_GPU)
```

Comment:
The multiple CMake workarounds introduced by me and Carl to get CI to pass are making the git history quite hard to follow.